docs/qcfractal/source/managers_config_api.rst

```diff
@@ -0,0 +1,53 @@
+Queue Manager API
+=================
+
```

Comment:
The top of this file needs some explanation of what this is and an example of how to use it. For example under cluster it might show:

```
cluster:
  option1: ...
```

I didn't understand this was a YAML mapping for a bit.